x-pack/plugins/enterprise_search/public/applications/index.tsx

```diff
@@ -68,12 +70,16 @@ export const renderApp = (
 
   ReactDOM.render(
     <I18nProvider>
-      <Provider store={store}>
-        <Router history={params.history}>
-          <App {...initialData} />
-          <Toasts />
-        </Router>
-      </Provider>
+      <EuiThemeProvider>
+        <KibanaContextProvider services={{ ...core, ...plugins }}>
```

Comment:
Thanks for the context and I agree with your assessment.

Reply:
It's fairly clearly stated in the [LogStream documentation](https://github.com/elastic/kibana/blob/master/x-pack/plugins/infra/public/components/log_stream/log_stream.stories.mdx#prerequisites), which is linked to in a docblock our own EntSearchLogStream file.

While I'm generally for comments, I do think there's a balance to be struck between over-commenting and under-commenting. Generally, I think that balance is helped by documentation files and additionally clear git/commit history. For instance, if someone wants to see why and when we added these providers/wrappers, all they have to do is go to the git blame/history for this line and they'll come find this commit/PR which has a ton of context and links.

In this specific scenario, I don't think we should add a comment that indicates these 2 providers are **only** for LogStream. It falls out of date quickly if we start using the providers and their context/hooks for non-LogStream use cases in the future. I also don't think we should add a note that says "we could start using this in the future". That type of comment would also fall out of date and stop making sense if we did start using it, or we could end up never using these providers except for LogStream and they just chill out there (perfectly reasonable).

I could definitely be biased however towards history in PRs (which I tend to reference back to semi-frequently), but I think it's both reasonable and easier to have the context for this be tied to a very specific point in time (i.e., this PR comment) than attempt to write an evergreen code comment for a future or direction we're not sure about. I did think about leaving a comment on `useKibana()` in KibanaLogic itself (or near where KibanaLogic is mounted), but ended up deciding it would be too long and would make more sense as a PR comment instead. 🤷